src/agent/agent.c

```diff
@@ -652,6 +652,48 @@ static AgentUnitInfo *agent_ensure_unit_info(Agent *agent, const char *unit) {
         return info;
 }
 
+static int agent_method_passthrough_to_systemd_cb(
```

Comment:
trivial: would be nice to have it as "callback" instead of cb as you did some lines below. Anywya, LGTM.

Reply:
In this file all callback functions are post-fixed with `cb` while in `node.c` they are post-fixed with `callback`. So, I aligned at the file level. If we want to have one convention, I guess we should do it an a separate PR